components/Events/Filters.js

```diff
@@ -0,0 +1,204 @@
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState, React } from 'react';
+import events from '../../data/event';
+import styles from '../../styles/components/Events/Filters.module.scss';
+
+export default function filters(props) {
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+  const [chosenComms, setChosenComms] = useState({
+    studio: false,
+    icpc: false,
+    design: false,
+    cyber: false,
+    teachLA: false,
+    w: false,
+    ai: false,
+    hack: false,
+    board: false,
+  });
+  const [chosenDays, setChosenDays] = useState({
+    Sunday: false,
+    Monday: false,
+    Tuesday: false,
+    Wednesday: false,
+    Thursday: false,
+    Friday: false,
+    Saturday: false,
+  });
+  const [fromTime, setFromTime] = useState('');
+  const [toTime, setToTime] = useState('');
+  const [chosenLoc, setChosenLoc] = useState({
+    Online: false,
+    In_person: false,
+  });
+
+  const handleSearch = () => {
+    let allEvents = [...events];
+    //Check search input
+    const lowerCaseSearch = searchValue.toLowerCase();
+    allEvents = allEvents.filter((item) => {
+      if (String(item.title).toLowerCase().includes(lowerCaseSearch))
+        return true;
+      if (String(item.committee).toLowerCase().includes(lowerCaseSearch))
+        return true;
+      if (String(item.location).toLowerCase().includes(lowerCaseSearch))
+        return true;
+      if (String(item.description).toLowerCase().includes(lowerCaseSearch))
+        return true;
+      return false;
+    });
```

Comment:
Nice job on all the functional methods!! 

We can do something like ;)
```suggestion
    const lowerCaseSearch = searchValue.toLowerCase();
    let allEvents = events.filter((item) =>
      [item.title, item.committee, item.location, item.description].some(
        (itemsToSearch) =>
          String(itemsToSearch).toLowerCase().includes(lowerCaseSearch)
      )
    );
```